libnmstate/state.py

```diff
@@ -32,6 +32,68 @@
 from libnmstate.schema import Interface
 from libnmstate.schema import InterfaceState
 from libnmstate.schema import InterfaceType
+from libnmstate.schema import Route
+
+
+class RouteEntry(object):
+
+    # Mapping dict key name to RouteEntry property name
+    KEY_PROPETY_MAP = {
```

Comment:
There is a typo in the constant, an `R` is missing. Correct would be `KEY_PROPERTY_MAP`.